lib/db/DB.ts

```diff
@@ -103,6 +107,18 @@ class DB {
       await this.sequelize.query('SET FOREIGN_KEY_CHECKS = 1', null, options);
     });
   }
+
+  async dbCreate() {
+    try {
+      this.sequelize = new Sequelize('', this.sequelize.options.username, this.sequelize.options.password, this.sequelize.options);
+      await this.sequelize.authenticate();
+      await this.sequelize.query(`CREATE DATABASE ${this.sequelize.options.database};`);
+      await this.sequelize.query(`USE ${this.sequelize.options.database}`);
+    } catch (err) {
+      this.logger.error('unable to connect to the database', err);
```

Comment:
this will apply to a `CREATE DATABASE` error as well, which will end in a misleading error message.

Reply:
agree refactored them with proper error messages.